doc/adr/0003-directory-structure.md

```diff
@@ -0,0 +1,39 @@
+# 3. Define directory structure
+
+Date: 2020-12-01
+
+## Status
+
+Accepted
+
+## Context
+
+The will be many services contributed by multiple people. We need a common way of structuring the services
+so it's easy for the user to pick up any one service.
+
+## Decision
+
+Each service will be its own library. All services will be in the services directory.
+Each service will contain at least an include folder and a mbed_lib.json.
+
+An example service called `example` would look like this:
+
+```
+services/example/include/ble-service-example/example.h
```

Comment:
if the example service is named example, shouldn't it be `services/example/include/example/example.h` ?

Reply:
The prefix is to avoid clashes with non ble services. The library name is called ble-service-example. The directory name for the include is the same as library name.